Environmental Sensing/Radon Sensor Signoff.md

```diff
@@ -0,0 +1,75 @@
+
+The Radon Eye RD200 was selected by the team due to its compact and lightweight nature, distinguishing it from other radon sensors. It maintains dependable precision and accuracy when detecting radon gas levels. This guarantees the acquisition of reliable data for evaluating potential health hazards.
+
+# Constraints:
+
+One important constraint is Sensor Accuracy and Reliability. The radon sensor is crucial for accurately detecting and measuring radon levels. It must provide precise readings to assess the concentration of radon gas effectively. To ensure the radon sensor is accurate and reliable, there are several requirements. The sensor needs to be able to detect 0.5 counts per minute (cpm) per picocurie per liter (pCi/l) at a radon level of 10 pCi/l (or 30 counts per hour per picocurie per liter). It should operate within a temperature range of 10 to 40 degrees Celsius and with a relative humidity below 90%. The measurement range should be between 0.1 and 99.99 pCi/l. The precision should be within 10% at 10 pCi/l, and the accuracy should be within ±10%, with an error of less than ±0.5 pCi/l. The sensor should provide the first reliable data output in under 60 minutes and update the data every 10 minutes using a 60-minute moving average. The data log should store up to one year's worth of data, with intervals of 1 hour.
+
+The constraint for power consumption is that the radon sensor, specifically the RD200 model, should not consume more than 5 Volts (V). However, the RD200 model operates at DC 12 ± 0.1V, 65mA, which exceeds the desired power consumption limit. Therefore, if we want to conform to the power constraint, modifications or adaptations will be needed to reduce the RD200's power consumption to no more than 5V.
+
+Size and Weight is also a consideration. The radon sensor (RD200) needs to be compact and lightweight so that it can easily be integrated into the robot. This allows the robot to maintain its mobility and functionality, even in small crawl space areas. The RD200 sensor with its dimensions of 80mm x 120mm and a weight of 240g should be accommodated.
+
+
+The Integration Complexity, while connecting the Radon Eye RD200 is relatively simple, integrating the sensor into the robot's data collection and processing system may pose challenges. It is important to carefully consider the robot's architecture and ensure compatibility with the sensor. Additionally, the sensor should be able to communicate with the microcontroller using protocols like Serial Peripheral Interface (SPI) and Inter-Integrated Circuit (I2C).
```

Comment:
This seems true. But we need to know what the constraints are regarding integrating the sensor.